packages/cf-component-wizard/src/Wizard.js

```diff
@@ -0,0 +1,115 @@
+const React = require('react');
+const {PropTypes} = React;
+
+const Progress = require('cf-component-progress');
+const {Button} = require('cf-component-button');
+const {
+  Modal,
+  ModalHeader,
+  ModalTitle,
+  ModalClose,
+  ModalBody,
+  ModalFooter,
+  ModalActions
+} = require ('cf-component-modal');
+
+class Wizard extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.handleNextClick = this.handleNextClick.bind(this);
+    this.handleBackClick = this.handleBackClick.bind(this);
+  }
+
+  // The Progress component expectes a similar set of steps
+  // with a few less fields
+  extractProgressSteps(wizardSteps) {
+    return wizardSteps.map((step) => {
+      const {id, label, disabled} = step;
+      return {id, label, disabled};
```

Comment:
Technically, we kinda don't. I purposely made the steps data structure match between wizard and progress, except that the wizard's steps has more things. I figured people might dislike variables leaking into progress, so I added this.